client/src/pages/ProfileDetail/ProfileDetail.tsx

```diff
@@ -0,0 +1,113 @@
+import { Avatar, Card, CircularProgress, Grid, Typography } from '@mui/material';
+import React, { useEffect, useState } from 'react';
+import { useStyles } from './useStyles';
+import { LocationOn } from '@mui/icons-material';
+import RequestForm from '../../components/RequestForm/RequestForm';
+import { FormikHelpers } from 'formik';
+import { makeBooking } from '../../helpers/APICalls/bookingInfo';
+import { BookingInfo } from '../../interface/BookingInfo';
+import { useAuth } from '../../context/useAuthContext';
+import { loadProfile } from '../../helpers/APICalls/loadProfile';
+import { Profile } from '../../interface/Profile';
+import ProfileGallery from '../../components/ProfileGallery/ProfileGallery';
+
+const ProfileDetail = (): JSX.Element => {
+  const classes = useStyles();
+  const { loggedInUser } = useAuth();
+  const [profile, setProfile] = useState<Profile>();
+
+  useEffect(() => {
+    !profile &&
+      loadProfile('6204aa0e6d1e83d0824f9b17').then((data) => {
+        if (data.success) {
+          console.log(data.success);
+          setProfile(data.success.profile);
+        }
+      });
+  }, [profile]);
+
+  const handleSubmit = (
+    {
+      dropInDate,
+      dropInTime,
+      dropOffDate,
+      dropOffTime,
+    }: { dropInDate: Date; dropInTime: string; dropOffDate: Date; dropOffTime: string },
+    { setSubmitting }: FormikHelpers<{ dropInDate: Date; dropInTime: string; dropOffDate: Date; dropOffTime: string }>,
+  ) => {
+    if (dropInTime.includes('pm')) {
+      dropInDate.setHours(parseInt(dropInTime) + 12, 0, 0, 0);
+    } else {
+      dropInDate.setHours(parseInt(dropInTime), 0, 0, 0);
+    }
+    if (dropOffTime.includes('pm')) {
+      dropOffDate.setHours(parseInt(dropOffTime) + 12, 0, 0, 0);
+    } else {
+      dropOffDate.setHours(parseInt(dropOffTime), 0, 0, 0);
+    }
+
+    if (loggedInUser && profile) {
+      const bookingInfo: BookingInfo = {
+        _id: undefined,
+        petOwner: loggedInUser,
+        sitter: profile.userId,
+        startDate: dropInDate,
+        endDate: dropOffDate,
+        status: 'pending',
+        paid: true,
+      };
+
+      makeBooking(bookingInfo);
+      setSubmitting(false);
+    } else {
+      // add a snake bar for request login
+    }
```

Comment:
Remove clause if not used

Reply:
@JunZheng007 Please address comment above (I would suggest disabling the 'send request' button if a user's not logged in 👍🏽

Reply:
Apply it on RequestForm.tsx